cloud_info_provider/providers/mesos.py

```diff
@@ -95,7 +95,12 @@ def _update(old, new):
         # gather & update dynamic info
         for api_endp in self.api_endpoints:
             api_url = "/".join([self.framework_url, api_endp])
-            r = requests.get(api_url, headers=self.headers, verify=self.cacert)
+            r = requests.get(
+                api_url,
+                headers=self.headers,
+                verify=self.cacert,
+                timeout=self.opts.timeout,
+            )
```

Comment:
From where come those changes?

Reply:
this was because bandit was complaining about not having timeout in request, so fixing tests